project/src/components/room-page/room-page.jsx

```diff
@@ -20,171 +29,86 @@ function RoomPage(props) {
-                    <li className="property__inside-item">Dishwasher</li>
-                    <li className="property__inside-item">Cabel TV</li>
-                    <li className="property__inside-item">Fridge</li>
-                  </ul>
-                </div>
-                <div className="property__host">
-                  <h2 className="property__host-title">Meet the host</h2>
-                  <div className="property__host-user user">
-                    <div className="property__avatar-wrapper property__avatar-wrapper--pro user__avatar-wrapper">
-                      <img className="property__avatar user__avatar" src="img/avatar-angelina.jpg" width="74" height="74" alt="Host avatar" />
+
+          {offers.map((offer) => {
+            const link = `${AppRoute.OFFER_$ID}-${offer.id}`;
+            if (link === location.pathname) {
+              return (
+                <section className="property">
+                  <div className="property__gallery-container container">
+                    <div className="property__gallery">
+                      {offer.images.map((images) => (
+                        <div key={images} className="property__image-wrapper">
+                          <img className="property__image" src={images} alt="Photo studio" />
+                        </div>
+                      ))}
                     </div>
-                    <span className="property__user-name">Angelina</span>
-                    <span className="property__user-status">Pro</span>
                   </div>
-                  <div className="property__description">
-                    <p className="property__text">
-                      A quiet cozy and picturesque that hides behind a a river by the unique lightness of Amsterdam. The building is green and from 18th century.
-                    </p>
-                    <p className="property__text">
-                      An independent House, strategically located between Rembrand Square and National Opera, but where the bustle of the city comes to rest in this alley flowery and colorful.
-                    </p>
-                  </div>
-                </div>
-                <section className="property__reviews reviews">
-                  <h2 className="reviews__title">Reviews &middot; <span className="reviews__amount">1</span></h2>
-                  <ul className="reviews__list">
-                    <li className="reviews__item">
-                      <div className="reviews__user user">
-                        <div className="reviews__avatar-wrapper user__avatar-wrapper">
-                          <img className="reviews__avatar user__avatar" src="img/avatar-max.jpg" width="54" height="54" alt="Reviews avatar" />
+                  <div className="property__container container">
+                    <div className="property__wrapper">
+                      {offer.isPremium ? <div className="property__mark"><span>Premium</span></div> : ''}
```

Comment:
```suggestion
                      {offer.isPremium && <div className="property__mark"><span>Premium</span></div>}
```
Как вариант. Так даже лучше, более наглядней